template/metadata.yaml.tpl

```
@@ -1,8 +1,8 @@
 service:
-  full_name: "{{ .ServiceFullName }}"
-  short_name: "{{ .ServiceID }}"
+  full_name:
+  short_name:
```

I think it can be useful to fill in this value for service team by default. and provide them instruction to update it here and also in READ_BEFORE_COMMIT.md
```suggestion
  full_name: "{{ .ServiceFullName }}" [Update Accordingly]
  short_name: "{{ .ServiceID }}" [Update Accordingly]
```

---

I'm very scared to do that because I don't trust that people will have a high bar for this sort of detail. This is something I hope we will catch in the code review, but the casing is pretty important here. These strings are used in the documentation and some of the more user-facing parts of the system, so we need to keep them consistent and "pretty".